<launch>
  <node pkg="multilayer_object_tracking" type="point_cloud_image_creator_node"
	name="point_cloud_image_creator" output="screen">
    <remap from="input_info" to="/camera/depth_registered/camera_info" />
    <remap from="input" to="/selected_pointcloud" />
    <remap from="input_image" to="/camera/rgb/image_rect_color"/>
  </node>
  <node pkg="image_view" type="image_view"
	name="image_view" output="screen" >
    <remap from="image" to="/cloud_image/output/image" />
  </node>
</launch>
